sky/spot/controller.py

```diff
@@ -137,8 +137,13 @@ def _run(self):
                     status_to_set = spot_state.SpotStatus.FAILED
                     if job_status == job_lib.JobStatus.FAILED_SETUP:
                         status_to_set = spot_state.SpotStatus.FAILED_SETUP
+                    failure_reason = (
+                        'To see the details, run: '
+                        f'sky spot logs --controller {self._job_id}')
```

Comment:
Should we update everywhere, e.g., `_exec_code_on_head`'s `finally`, to not print `sky logs <controller> ..` anymore?